src/server/repositories/UserRepository.ts

```diff
@@ -90,39 +93,82 @@ export class UserRepository implements UserRepositoryInterface {
     conditions: UserUrlsQueryConditions,
```

Comment:
This file looks much neater! Thanks for refactoring the query conditions for retrieving a url, this is much more maintainable.